app/Console/Commands/BudgetInstall.php

```diff
@@ -44,13 +49,26 @@ public function artisanPrepare() {
     }
 
     public function yarnBuild() {
-        $this->executeCommand(['yarn', 'run', 'development']);
+        $command = ['yarn', 'run', 'development'];
+        if ($this->option('dev')) {
+            array_push($command, '--watch');
+        }
+
+        $this->executeCommand($command);
+    }
+
+    public function generateVuei18nTranslations() {
+        $this->executeCommand(['php', 'artisan', 'vue-i18n:generate']);
     }
 
     public function handle() {
-        $this->composerInstall();
-        $this->yarnInstall();
-        $this->artisanPrepare();
-        $this->yarnBuild();
+        $this->generateVuei18nTranslations();
+
+        if (!$this->option('translation')) {
+            $this->composerInstall();
```

Comment:
Wait so, if I **do** use the translation flag, I **cannot** install Composer dependencies and shit like that?

Reply:
Yes, because if you use the translation-flag it will **only** generate the translations. If you run budget:install, it will install everything